src/style.css

```diff
@@ -0,0 +1,83 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+.html {
+  font-family: Avenir, sans-serif;
+}
+
+.input::placeholder {
+  font-family: inherit;
+  font-weight: 300;
+  color: #3d4e61;
+  font-size: 16px;
+}
+
+.inputSmall::placeholder {
+  font-size: 14px;
+}
+
+.input {
+  width: 100%;
+  height: 70px;
+  padding-left: 62px;
+  box-sizing: border-box;
+
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  outline-style: none;
+
+  background: url(./images/Search.svg) 25px 24px no-repeat #fff;
+
+  font-family: inherit;
+  font-size: 16px;
+}
+
+.input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+
+  background-image: url(./images/Search.svg);
+  background-position: 25px 24px;
+  background-repeat: no-repeat;
+  background-color: #fff;
+}
+
+.input:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+
+  background-image: url(./images/Search.svg);
+  background-position: 25px 24px;
+  background-repeat: no-repeat;
+}
```

Comment:
why do you need to set same background image on hover/focus? don't duplicate it, you already have this in `.input`

Reply:
On hover it works without duplicating background image. But on focus image dissappearing without duplicating it. I guess it's because of this `.input:focus {
  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);` How can I fix this? @ivanchuk-yurii